### PR TITLE
Create responsive for header section

### PR DESCRIPTION
**Pull request by CBinh710**
Task: [Link](https://github.com/users/CBinh710/projects/4/views/1?pane=issue&itemId=74869450
- Mobile screen (width: 428px)
image
- Desktop screen (width: 1440px)
image

### CODE REVIEW

File: practice-one/src/sections/header.html [hunk 1]
```diff
@@ -13,4 +13,9 @@
     "text": "Download App"
     }'></include>
   </nav>
+  <button class="navbar-button">
```
Comment (hungdo-agilityio) [inline practice-one/src/sections/header.html:16]: div inside button seem wrong please check

File: practice-one/src/styles/sections/layout.css [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 
 html, body {
   font-family: var(--font-family-primary);
 }
+
+@media (max-width: 429px) {
+  .container {
+    padding: 24px;    
+  }
+}
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/sections/layout.css]: What about screen tablet?

Comment (CBinh710) [reply practice-one/src/styles/sections/layout.css]: In original screen don't have the design for tablet screen

File: practice-one/src/sections/header.html (hunk 1 above)
Comment (CBinh710) [reply practice-one/src/sections/header.html:16]: I changed it to span tag , thanks for your contribution

File: practice-one/src/styles/sections/layout.css (hunk 2 above)
Comment (hungdo-agilityio) [reply practice-one/src/styles/sections/layout.css]: I see, but we need to make sure everything is not broken right? Simply put a screenshot for tablet
This is your target:
```md
Create a responsive website :
Desktop breakpoint (1440px): Match with design
Mobile breakpoint(428px): Match with design
Another breakpoints: Avoid to break application
```

Comment (CBinh710) [reply practice-one/src/styles/sections/layout.css]: I changed it ,thanks

File: practice-one/src/sections/header.html [hunk 3]
```diff
@@ -13,4 +13,9 @@
...
+    <span class="navbar-icon-wrapper">
```
Comment (hungdo-agilityio) [inline practice-one/src/sections/header.html:17]: Do we need this span?

File: practice-one/src/styles/sections/header.css [hunk 4]
```diff
@@ -25,3 +26,52 @@
   color: var(--color-dark);
   text-decoration: none;
 }
+
+.navbar-button {
```
Comment (lintran-agilityio) [inline practice-one/src/styles/sections/header.css:30]: can create common class: display: none; to re-use

File: practice-one/src/styles/sections/header.css [hunk 5]
```diff
@@ -25,3 +26,52 @@
...
+  .logo-company-container {
+    width: 97px;
+    height: 62px;
+  }
+
+  .navbar-container {
+    gap: 20px;
+  }
+
+  .navbar-container > li > a,
```
Comment (lintran-agilityio) [inline practice-one/src/styles/sections/header.css]: should create class to modify, avoid use > tag

File: practice-one/src/styles/sections/header.css [hunk 6]
```diff
@@ -25,3 +26,52 @@
...
+  }
+
+  .navbar-button {
+    display: block;
+    border: none;
+    background-color: transparent;
+  }
+
+  .wrapper-navbar-container {
+    display: none;
```
Comment (lintran-agilityio) [inline practice-one/src/styles/sections/header.css]: the same

**Pull request merged**